src/main/java/PlayingLinkedQueue.java

```diff
@@ -0,0 +1,66 @@
+import java.util.NoSuchElementException;
+
+public class PlayingLinkedQueue {
```

Comment:
This being in the default package is kinda confusing. There's lots of playground code to release spanning various topics. It may make more sense to place this in a `queues` package, mirroring the topic name structure.

Reply:
Such is true for a lot of the classes. So maybe throw this isn't a "I'll think about it" issue. It'd mostly be for your own organization, but be a little harder for students as they'd have to make note of the package the class is in.

Reply:
_I'll think_ on the _I'll think about it_. 

I was considering making use of packages more throughout the course anyways as I think it may help with assignment content delivery.

Reply:
> _I'll think_ on the _I'll think about it_.

That means make an issue either way.

Reply:
#756 goes into this. If/when this change occurs, it will be a separate PR to do an overhaul reorganization.